ext/winevt/winevt_bookmark.c

```diff
@@ -1,18 +1,22 @@
 #include <winevt_c.h>
 
-static void bookmark_free(void *ptr);
-
-static const rb_data_type_t rb_winevt_bookmark_type = {
-  "winevt/bookmark", {
-    0, bookmark_free, 0,
-  }, NULL, NULL,
-  RUBY_TYPED_FREE_IMMEDIATELY
-};
+static void
+bookmark_free(void* ptr);
+
+static const rb_data_type_t rb_winevt_bookmark_type = { "winevt/bookmark",
+                                                        {
```

Comment:
It seems too deep indent.
Can you tweak the clang-format's option to break after the brace `{` ?
If it's impossible, I don't prevent to merge it.

Reply:
We can't find newline after brace settings.... :confounded: